lib/matplotlib/widgets.py

```diff
@@ -1572,19 +1729,19 @@ def __init__(self, ax, onselect, drawtype='box',
 
         if drawtype == 'box':
             if rectprops is None:
-                rectprops = dict(facecolor='white', edgecolor='black',
-                                 alpha=0.5, fill=False)
+                rectprops = dict(facecolor='red', edgecolor='black',
+                                 alpha=0.2, fill=True)
+            rectprops['animated'] = useblit
             self.rectprops = rectprops
-            self.to_draw = Rectangle((0, 0),
+            self.to_draw = self._shape_klass((0, 0),
                                      0, 1, visible=False, **self.rectprops)
             self.ax.add_patch(self.to_draw)
         if drawtype == 'line':
             if lineprops is None:
                 lineprops = dict(color='black', linestyle='-',
                                  linewidth=2, alpha=0.5)
+            lineprops['animated'] = useblit
```

Comment:
shouldn't this be `self.useblit` so that the correct value is used in case the canvas doesn't support blitting?